Automattic/wp-calypso
# Update/plan features grid shuffle #2

#### Proposed Changes

This reopens #66617 and fixes the error which prompted the revert in #66684.
The issue was caused by the feature list of the Free plan not having the updated `getPlanCompareFeaturesV2`. 
The fix involves removing the new function and passing in the experiment and making the switch in the `getPlanCompareFeatures` function.
This PR also updates the experiment name `pricing_packaging_plans_page_quick_improvements_v2`

#### Testing Instructions

* Follow the instructions from #66617

* Additionally, visit `/start/domains` while using the `treatment` variation
* Search and select a Free domain
* Once you're landed `/start/plan` there page should fine, without any related errors
<img width="320" alt="Screenshot on 2022-08-18 at 15-19-01" src="https://user-images.githubusercontent.com/2749938/185393009-e8569417-d893-4757-a9de-bde5ed9dd38e.png">
